lib/layers/data/models/remoute/personal_api_data/personal_data_api.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:json_annotation/json_annotation.dart';
+
+import '../../../../domain/entities/personal_data.dart';
+import 'data.dart';
+import 'Meta.dart';
+
+class PersonalDataApi {
+  @JsonKey(name: 'data')
+  final Map<String,Data>? data;
+  @JsonKey(name: 'meta')
+  final Meta? meta;
+  @JsonKey(name: 'status')
+  String status;
+
+  PersonalDataApi(
+      {required this.data, required this.meta, required this.status});
+
+  factory PersonalDataApi.fromJson(Map<String, dynamic> json) {
+    return PersonalDataApi(
+      data: json['data'] != null ? <String,Data>{
+        json['data'].keys.first : Data.fromJson(json['data'][json['data'].keys.first]),
+      } : null,
+      meta: json['meta'] != null ? Meta.fromJson(json['meta']) : null,
+      status: json['status'],
+    );
+  }
+
+  Map<String, dynamic> toJson() {
+    final Map<String, dynamic> data = <String, dynamic>{};
+    data['status'] = status;
+    data['data'] = this.data!;
+    if (meta != null) {
+      data['meta'] = meta?.toJson();
+    }
+    return data;
+  }
+
+  List<PersonalData> toList() {
+    List<PersonalData> result = [];
+    if (data?[data?.keys.first] == null) return result;
+    result.add(PersonalData(
+        dataTitle: 'clanId', dataValue: data![data?.keys.first]!.clanId.toString()));
```

Comment:
It not cool to se such amount of bang operators.
If you extract data?[data?.keys.first] to local variable dart will understand that it is not null here and you will not need to write ! every time. 
A